patches/net/minecraft/data/registries/RegistriesDatapackGenerator.java.patch

```diff
@@ -33,11 +41,36 @@
                              .flatMap(
                                  p_256552_ -> this.dumpRegistryCap(p_255785_, p_326736_, dynamicops, (RegistryDataLoader.RegistryData<?>)p_256552_).stream()
                              )
-@@ -54,6 +_,7 @@
+@@ -48,19 +_,21 @@
+         CachedOutput p_256502_, HolderLookup.Provider p_256492_, DynamicOps<JsonElement> p_256000_, RegistryDataLoader.RegistryData<T> p_256449_
+     ) {
+         ResourceKey<? extends Registry<T>> resourcekey = p_256449_.key();
++        com.mojang.serialization.Codec<Optional<net.neoforged.neoforge.common.conditions.WithConditions<T>>> conditionalCodec = net.neoforged.neoforge.common.conditions.ConditionalOps.createConditionalCodecWithConditions(p_256449_.elementCodec());
```

Comment:
Please make this a var.

Reply:
Apologies - wasn't sure on the convention for this. Resolved :)